CHANGELOG.md

```diff
@@ -1,94 +1,155 @@
-[unreleased]
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+Note that in line with [Django REST Framework policy](http://www.django-rest-framework.org/topics/release-notes/),
+any parts of the framework not mentioned in the documentation should generally be considered private API, and may be subject to change.
+
+## [Unreleased]
+
+### Added
 
 * Add testing configuration to `REST_FRAMEWORK` configuration as described in [DRF](https://www.django-rest-framework.org/api-guide/testing/#configuration)
 * Add `HyperlinkedRelatedField` and `SerializerMethodHyperlinkedRelatedField`. See [usage docs](docs/usage.md#related-fields)
 * Add related urls support. See [usage docs](docs/usage.md#related-urls)
-* Replaced binary `drf_example` sqlite3 db with a [fixture](example/fixtures/drf_example.yaml). See [getting started](docs/getting-started.md#running-the-example-app).
+* Add optional [jsonapi-style](http://jsonapi.org/format/) filter backends. See [usage docs](docs/usage.md#filter-backends)
+
+### Changed
+
+* Replaced binary `drf_example` sqlite3 db with a [fixture](example/fixtures/drf_example.json). See [getting started](docs/getting-started.md#running-the-example-app).
+
+### Deprecated
+
```

Comment:
I believe we've decided to yank this PR before a release ever happened, so I don't think it needs to be listed as deprecated.

Reply:
Fair enough. Removed.